symengine/ruby/callgrind_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+valgrind --tool=callgrind --dump-instr=yes --simulate-cache=yes --collect-jumps=yes ruby -S rspec spec
```

Comment:
This file should not be here. You can put it into `bin` directory (in the root).